modules/oe_whitelabel_starter_event/config/install/core.entity_view_display.node.oe_sc_event.oe_w_content_banner.yml

```diff
@@ -27,6 +29,18 @@ content:
     third_party_settings: {  }
     weight: 1
     region: content
+  oe_sc_event_registration_url:
+    type: event_link_external
```

Comment:
the formatter should be defined in oe_whitelabel_helper and then its machine name will be different